core/src/main/kotlin/com/tschuchort/compiletesting/SourceFile.kt

```diff
@@ -9,7 +9,7 @@ import org.intellij.lang.annotations.Language
  * A source file for the [KotlinCompilation]
  */
 abstract class SourceFile {
-    internal abstract fun writeIfNeeded(dir: File): File
+    internal abstract fun writeTo(dir: File): File
```

Comment:
I figured that renaming this makes sense since it will always write the contents now. Can revert though.